deme/tests/expand-demes-testold.metta

```diff
@@ -36,6 +36,7 @@
 !(import! &self metta-moses:scoring:bscore)
 !(import! &self metta-moses:scoring:fitness)
 !(import! &self metta-moses:scoring:complexity-based-scorer)
+!(import! &self metta-moses:scoring:cacheSpace)
```

Comment:
@eyuuab If this file is working, can we rename the test file so that it gets picked up by the script?